src/Riok.Mapperly.Abstractions/MapperDefaultsAttribute.cs

```diff
@@ -7,4 +7,12 @@ namespace Riok.Mapperly.Abstractions;
 /// </summary>
 [AttributeUsage(AttributeTargets.Assembly)]
 [Conditional("MAPPERLY_ABSTRACTIONS_SCOPE_RUNTIME")]
-public sealed class MapperDefaultsAttribute : MapperAttribute;
+public sealed class MapperDefaultsAttribute : MapperAttribute
+{
+    /// <summary>
+    /// Controls the priority of constructors used in mapping.
+    ///  When enabled, a parameterless constructor is prioritized over constructors with parameters.
+    ///  When disabled, accessible constructors are ordered in descending order by their parameter count.
+    /// </summary>
+    public bool PreferParameterlessConstructors { get; set; } = true;
```

Comment:
As this extends the `MapperAttribute` this change can be reverted (the property is already inherited).

Reply:
I will remove it .
 but please, could you give more explanation about  (the property is already inherited) 
why the class MapperDefaultsAttribut is there while it contains nothing ?

Reply:
The idea behind the separate `MapperDefaultsAttribute`  class is that we want to keep the option to introduce properties exclusive to `MapperDefaultsAttribute` or diverging properties without introducing a breaking change.